src/include/miopen/solver/conv_direct_naive_conv.hpp

```diff
@@ -53,5 +58,157 @@ bool IsOutputBfp16(const ProblemDescription&);
 bool IsOutputInt8(const ProblemDescription&);
 bool IsOutputInt32(const ProblemDescription&);
 
+int GetGroupStrideIndex(const ProblemDescription& problem);
+
+void printTensorStrides(const TensorDescriptor& inDesc,
```

Comment:
Do not litter `miopen::solver` namespace with solver-specific functions.
```suggestion
void ConvDirectNaivePrintTensorStrides(const TensorDescriptor& inDesc,
```

Reply:
This is a debug utility function and I would like to move this to a better file. I'm open to suggestions.

Reply:
The comment is not about the proper file. It's about the global namespace. `miopen::solver::printTensorStrides()` seems too generic because it is intended to work with convolutions only. All the more this function was used in Naive Convolution solvers only. That's why I asked to add a prefix, which seemed as a simplest solution.

If you would like to emphasize it's debugging purpose, then you can put it into the `miopen::debug` namespace (preferred) or add a `Debug` prefix.

Another simple solution is adding a method like `TensorsStridesToString()` into convolution problem description (and then using it in MIOPEN_LOG* macros when necessary).

It's all up to you, but `miopen::solver::printTensorStrides()` needs to be changed somehow.

Reply:
I think I'll move it to ConvDescriptor or ConvDataTensors class and have it return a string.

Reply:
On second thought, I'll add a prefix because moving it to `ProblemDescription` class is not right as the strides I intend to print are for the tensors passed to `Invoker` lambda which could be different from the ones in `ProblemDescription`. See https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2334#discussion_r1340438435